src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertPlaceholderToInterpolatedStringRefactoringProvider.cs

```diff
@@ -146,6 +148,28 @@ internal abstract class AbstractConvertPlaceholderToInterpolatedStringRefactorin
             return document.WithSyntaxRoot(newRoot);
         }
 
+        private SyntaxNode GetParamsArgument(SeparatedSyntaxList<TArgumentSyntax> arguments)
+        {
+            var i = arguments.IndexOf(argument => GetArgumentName(argument) == StringFormatArguments.ArgsArgumentName);
+            return i >= 0 ? arguments[i] : arguments[1];
+        }
+
+        private TArgumentSyntax GetFormatArgument(SeparatedSyntaxList<TArgumentSyntax> arguments)
+        {
+            var index = arguments.IndexOf(argument => GetArgumentName(argument) == StringFormatArguments.FormatArgumentName);
+            return index >= 0 ? arguments[index] : arguments[0];
+        }
+
+        private TArgumentSyntax GetArgument(SeparatedSyntaxList<TArgumentSyntax> arguments, int index)
+        {
+            if (arguments.Count > 4)
```

Comment:
if argument more than 4 it means that used the following overload ```String.Format(String format, params Object[] args)``` without using named arguments. So we 100% sure in that case that we have no named arguments.